jobbergate-api/tests/apps/job_submissions/test_routers.py

```diff
@@ -2619,3 +2619,157 @@ async def test_job_submissions_metrics__start_time_less_greater_than_end_time(
     mocked_session_execute.return_value.fetchall.assert_not_called()
     mocked_sa_text.assert_not_called()
     assert response.json() == {"detail": "End time must be greater than the start time."}
+
+
+@pytest.mark.parametrize(
+    "permission, num_rows",
+    [
+        (Permissions.ADMIN, 3),
+        (Permissions.JOB_SUBMISSIONS_READ, 8),
+    ],
+)
+async def job_submissions_metrics_timestamps__successful_request(
```

Comment:
Why don't these tests start with `test_`? Are they even being run as part of the testing suite?

Reply:
I forgot, thanks.

Reply:
check [9f2eaf0](https://github.com/omnivector-solutions/jobbergate/pull/683/commits/9f2eaf02e9dcce057ebc06c15627505180ae8210)